connectors/helpers/base/egr_connector.yaml

```diff
@@ -7,6 +7,7 @@ metadata:
   name: egr-connector
   labels:
     app: egr-connector
+    type: connector
```

Comment:
To avoid conflicts labels should include a prefix. For example `m4d.ibm.com/componentType`

Reply:
OK,